### PR TITLE
Add command to edit secret using an editor

### PR DESCRIPTION
**Pull request by jmcs**
Add ./cmd/edit-team-secret to edit a secret using the default $EDITOR.

### CODE REVIEW

**Review by thilp — COMMENTED** [review]
Thanks, I like it!

How do you suggest we deal with the various unencrypted backup files created by the arbitrary `$EDITOR`? For example [for vim](https://vim.fandom.com/wiki/Encryption):

> When using an external program, be certain to turn off options like persistent undo ([:help 'undofile'](http://vimdoc.sourceforge.net/cgi-bin/help?tag=%27undofile%27)), backup files ([:help 'backup'](http://vimdoc.sourceforge.net/cgi-bin/help?tag=%27backup%27)), swap files ([:help 'swapfile'](http://vimdoc.sourceforge.net/cgi-bin/help?tag=%27swapfile%27)), and saving certain information like register contents to the .viminfo file ([:help 'viminfo'](http://vimdoc.sourceforge.net/cgi-bin/help?tag=%27viminfo%27)), to prevent that the entire file or parts of the file are available on your disk unencrypted.

I imagine we can’t have a universal solution here (different editors will do different things, and I don’t expect efforts like [EditorConfig](https://editorconfig.org) to be sophisticated enough yet). Maybe detect the most common editors and inject the corresponding options? Or simply warn users in the README and let them deal with it (we can probably expect most of them won’t)?

**Comment by jmcs** [comment]
I can add a warning to the README and at least `rm $TMP_FILE* || true` in the cleanup to try to deal with the most common backup file conventions. WDYT?

**Comment by thilp** [comment]
That would be much appreciated :+1: Thanks!

File: cmd/edit-team-secret [hunk 1]
```diff
@@ -13,6 +13,8 @@ TMP_FILE=$(mktemp)
 function cleanup()
 {
     rm $TMP_FILE
+    # try to remove backup files
+    rm $TMP_FILE* 2>1 || true
```
Comment (thilp) [inline cmd/edit-team-secret]: `2>1` is redirecting stderr to a file named `1` :wink:

You probably meant `2>&1`, but I think `-f` is the way to go anyway (it [is posix](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/rm.html) after all):
> `-f`: Do not prompt for confirmation. Do not write diagnostic messages or modify the exit status in the case of no file operands, or in the case of operands that do not exist.

```suggestion
    rm -f $TMP_FILE*
```

Comment (thilp) [reply cmd/edit-team-secret]: And then we can probably also get rid of `|| true`, sorry to only notice now, will edit the suggestion above.

**Pull request merged**